packages/entities/entities-shared/src/components/entity-base-form/EntityBaseForm.vue

```diff
@@ -90,7 +90,9 @@
       @close="toggle()"
     >
       <div>
-        {{ t('baseForm.configuration.message') }}
+        <slot name="configuration-slideout-message">
```

Comment:
nit: should this wrapper div have a test-id or something so it can be asserted on in your tests?